db/migrations/0001_initial.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 4.0.2 on 2024-11-18 16:32
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Guild',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+                ('description', models.TextField(blank=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Race',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+                ('description', models.TextField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Skill',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.CharField(max_length=255, unique=True)),
+                ('bonus', models.CharField(max_length=255)),
+                ('race', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='db.race')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Player',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('nickname', models.CharField(max_length=255, unique=True)),
+                ('email', models.EmailField(max_length=255)),
+                ('bio', models.CharField(max_length=255)),
+                ('created_at', models.DateTimeField(auto_now=True)),
```

Comment:
The ForeignKey reference to 'db.guild' should be checked. If the models are in the same app, you can simplify the reference to just 'Guild'.